object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,21 @@
+/* exported defaults */
+/*
+   Go through each key found in source and:
+   Check if that key value matches any key value found in target
+   If it does NOT match any key then...
+   Add that key to target object and assign its value as the value associated with that key in source object BUT
+   If the values found are different but the keys exist in both object do not assign the source value to the target object
+*/
+function defaults(target, source) {
+  var targetKeys = Object.keys(target);
```

Comment:
Try to do this without turning the object into an array 🤓

Reply:
![image](https://user-images.githubusercontent.com/68949703/214432099-f11703c7-371a-4aac-ab56-df33303b23fb.png)
![image](https://user-images.githubusercontent.com/68949703/214432124-48e515db-d1b4-4705-a78c-73b8ade4081f.png)
Ended up shortening it by quite a bit!